### ISSUE TITLE
Peek editor tree color tokens are broken

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#64829]
Set color customizations for the peek widget:

```
"workbench.colorCustomizations": {
        "peekViewResult.selectionBackground": "#ff0000",
        "peekViewResult.fileForeground": "#00ff00",
        "peekViewResult.selectionForeground": "#0000ff"
    }
```

1.29:
![image](https://user-images.githubusercontent.com/323878/49837300-bcba2580-fd5a-11e8-9a57-85d3cd176ec7.png)

1.30: 
![image](https://user-images.githubusercontent.com/323878/49837342-de1b1180-fd5a-11e8-8c58-f638f3ee6d3c.png)

Candidate or 1.30.1 maybe?

**Comment by jrieken** [comment]
Grrr, another regression... I am for putting this into 1.30.1

**Comment by jrieken** [comment]
I have pushed a fix to master and I have created https://github.com/Microsoft/vscode/pull/64880. @roblourens we think having for for `1.30.1` is good enough, otherwise the PR can be merged for `1.30`
